src/components/Assemblies/CabinetRole.svelte

```diff
@@ -1,21 +1,67 @@
 <script lang="ts">
+	import type { AssemblyMember } from '$lib/datasheets/assembly-member';
 	import type { CabinetSeat } from './shared';
 
 	export let cabinets: CabinetSeat[];
 	export let role: CabinetSeat['role'];
+
+	const show = (event: MouseEvent, member: AssemblyMember) => {
+		const memberName = `${member.firstname} ${member.lastname}`;
+		const tooltipElem = document.getElementById('tooltip');
+
+		if (tooltipElem) {
+			tooltipElem.setAttribute(
+				'style',
+				`left: ${event.layerX}px; top: ${event.layerY + 20}px; display: flex;`
+			);
+		}
+
+		const memberElem = document.getElementById('member-name');
+		const roleElem = document.getElementById('role');
+
+		if (memberElem) {
+			memberElem.textContent = memberName;
+		}
+
+		if (roleElem) {
+			roleElem.textContent = member.assemblyRole.role;
+		}
+	};
+
+	const hide = () => {
+		const tooltipElem = document.getElementById('tooltip');
+
+		if (tooltipElem) {
+			tooltipElem.setAttribute('style', `display: none;`);
+		}
+	};
 </script>
 
 <div class="role">
 	<p class="heading-compact-01">{role}</p>
 	<div class="group-dot">
 		{#each cabinets.find((c) => role === c.role)?.parties || [] as party}
-			{#each Array.from({ length: party.count || 0 }, (_, i) => i) as _}
-				<span class="dot" style="background-color: {party.color || '#8D8D8D'};"></span>
+			{#each party?.members ?? [] as member}
+				<!-- svelte-ignore a11y-no-static-element-interactions -->
```

Comment:
There is a reason why it's a warning. Static element like `span` or `div` was not designed to be interactive. But if we want to, in this case, provide a `role` attribute to explain its use case.

```html
<div role="tooltip">
```

Without this, blind people who use screen readers have no idea they can interact with a tooltip. (a11y stand for accessibility)

Reply:
Done.